app/src/main/java/org/fossasia/openevent/app/data/models/Ticket.java

```diff
@@ -19,7 +20,7 @@
     private long quantity;
     private String type;
 
-    @ForeignKey(stubbedRelationship = true)
+    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
```

Comment:
Actually just deleting event is giving error because the attendees and tickets are also linked to event.